src/app/core/entity-details/entity-details/entity-details.component.ts

```diff
@@ -91,8 +91,7 @@ export class EntityDetailsComponent implements EntityDetailsConfig, OnChanges {
     }
     if (changes.id) {
       this.loadEntity(this.id);
-    }
-    if (changes.panels) {
+    } else if (changes.panels) {
```

Comment:
```suggestion
    }
    if (changes.panels && !changes.id) {
      // on "id" changes, panels are automatically re-initialized already
```
as we are mostly using independent if-clauses, this might be more consistent and make the whole interdependence clear in the single line?